x/gov/abci.go

```diff
@@ -151,32 +169,40 @@ func processSecurityVote(ctx sdk.Context, k keeper.Keeper, proposal types.Propos
 		// the error message is logged.
 		err := handler(cacheCtx, proposal.GetContent())
```

Comment:
is the handler actually applied here? if it's security voting it shouldn't require handler being applied since it should be handled after regular voting period passes

Reply:
yes，the handler actually  is applied here.
Only `CertifierUpdateProposal`, other security voting will not execute here.